cerbero/packages/package.py

```diff
@@ -111,6 +111,8 @@ class PackageBase(object):
     resources_postinstall = 'postinstall'
     resources_postremove = 'postremove'
     conflicts = ''
+    replaces = ''
```

Comment:
Yes, those are debian specific. There is a equivalent in RPM. 'conflicts' is used also in RPM. Names of packages might differ for debian and RPM so maybe we should have separated set of variables for both?